zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -2542,5 +2542,4 @@ public boolean isReconfigEnabled() {
             return null;
         }
     }
-
```

Comment:
nit: unwanted diff

Reply:
Running Checkstyle gave these issues

Reply:
There must be a way to bypass them then. @hanm @eolivelli ?